DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts/EditSfsoLead.cshtml

```diff
@@ -0,0 +1,11 @@
+@page
+@model EditSfsoLeadModel
+
+@{
+  Layout = "_TrustLayout";
+}
+
+<form method="post">
```

Comment:
This form tag (and error summary include) is identical on both pages. If we move it inside `_EditContactsForm` then there will be less duplication and all the form will live in one file